changelog/default_after_variadic.dd

```diff
@@ -10,8 +15,10 @@ string log(T...)(T a, string file = __FILE__, int line = __LINE__)
 assert(log(10, "abc") == text(__FILE__, ":", __LINE__, " 10abc"));
 ---
 
-This should be preferred to the previous workaround, which caused template bloat:
+This should be preferred to the previous workaround, which causes a new template instantiation
+for every invocation:
 ---
 string log(string file = __FILE__, int line = __LINE__, T...)(T a);
 ---
 
+For more details, see $(DMDSRC test/runnable/testdefault_after_variadic.d).
```

Comment:
Should we really refer to internal tests in the compiler?

Reply:
This was previously in the title.

Reply:
I know.

Reply:
Why don't we refer to the spec instead?

Reply:
https://github.com/dlang/dlang.org/pull/2169